index.html

```diff
@@ -16,29 +16,24 @@
     <div class="container">
       <header>
         <!-- Nav Section -->
-        <nav class="menu">
+        <nav class="navbar">
           <a href="#" class="logo">My Logo</a>
 
-          <ul>
-            <li>
-              <a href="#" class="active" onclick="markMenuItem(this)"
-                >Portfolio</a
-              >
-            </li>
-            <li><a href="#" onclick="markMenuItem(this)">About</a></li>
-            <li><a href="#" onclick="markMenuItem(this)">Contact</a></li>
+          <ul class="nav-menu">
+            <li class="nav-item"><a href="#portfolio" class="nav-link">Portfolio</a></li>
```

Comment:
- [ ] When I click on the mobile menu **Portfolio** the correct part of the page was not displayed. Kindly scroll to the bottom of the page and click on this option. Nothing won't happen(the correct part will not be displayed). Kindly fix :wrench: the issue.